doc/examples/geo.rst

```diff
@@ -36,7 +36,7 @@ insert a couple of example locations:
   >>> result = db.places.insert_many([{"loc": [2, 5]},
   ...                                 {"loc": [30, 5]},
   ...                                 {"loc": [1, 2]},
-  ...                                 {"loc": [4, 4]}])  # doctest: +ELLIPSIS
+  ...                                 {"loc": [4, 4]}])
```

Comment:
This `# docktest: +ELLIPSIS` feature is cool. So it's just another approach to wildcard characters basically?

Reply:
Basically, yep